Analytics/Request/BlockingRequestHandler.cs

```diff
@@ -26,8 +26,8 @@ internal class HttpClient : WebClient
         protected override WebRequest GetWebRequest(Uri address)
         {
             WebRequest w = base.GetWebRequest(address);
-            if (Timeout.Milliseconds != 0)
-                w.Timeout = Timeout.Milliseconds;
+            if (Timeout.TotalMilliseconds != 0)
```

Comment:
@dailycoding why did this change?

Reply:
If we set timeout to 3s, Timeout.Milliseconds will be 0 while the Timeout.Seconds is 3.
But Timeout.TotalMilliseconds indicates 3000.

Reply:
Right, but isn't `TotalMilliseconds` the total timeout, not the current time remaining?